pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -136,6 +136,10 @@ func (nc *nodeConfig) Configure() error {
 	if err := nc.configureNetwork(); err != nil {
 		return errors.Wrap(err, "configuring node network failed")
 	}
+	// configure windows metrics exporter
+	if err := nc.configureMetrics(); err != nil {
```

Comment:
Wondering why `configureMetrics` is part of  the nc object.  Shouldn't it be part of `Windows` object? Or are you waiting for network to be configured, if that is case, you can expose configureMetrics directly there. I believe `nc` should have methods that need node object to be accessible.

Reply:
Waiting for network is not necessary. I'm not quite sure what you mean/where to move this though. There is a `ConfirgureWindowsExporter()` method on the `Windows` object that is called from` configureMetrics()` as part of configuring the node.

Reply:
@ravisantoshgudimetla is suggesting for this to become  part of `nc.Windows.Configure()` if it is not dependent on network configuration. If it is dependent on network configuration it will become `nc.Windows.ConfigureMetrics()`. 
I am +1 for doing this.